Logger/LogFactory.cs

```diff
@@ -2,10 +2,19 @@
 {
     public class LogFactory
     {
+        private string FileName;
```

Comment:
Fields should be underscore PascalCase.
```suggestion
        private string _FileName;
```